src/core/contracts/registry.ts

```diff
@@ -145,8 +172,14 @@ export const isMessageSignatureAuthorizedTo = async (
   blockTag?: ethers.providers.BlockTag,
   opts?: ConfigOpts
 ): Promise<boolean> => {
-  const reg = await resolveId(dsnpUserId);
-  if (!reg) throw MissingContract;
+  const registrations = await getDSNPRegistryUpdateEvents(
+    {
+      dsnpUserId,
+    },
+    opts
+  );
+  if (registrations.length === 0) throw new MissingRegistrationContractError(dsnpUserId);
```

Comment:
Do we want to throw in this case? I think just returning false would suffice.

Reply:
I'll change it in my current task: https://www.pivotaltracker.com/story/show/177926772